.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Wiki
+    about: 'Please visit the Citizens Wiki before reporting issues.'
+    url: 'https://wiki.citizensnpcs.co/Citizens_Wiki'
+  - name: Citizens API
+    about: 'Documentation about the API of Citizens.'
+    url: 'https://wiki.citizensnpcs.co/API'
+  - name: Development Builds
+    about: 'Dev builds of Citizens. DO NOT USE ON PRODUCTION SERVERS!'
```

Comment:
` DO NOT USE ON PRODUCTION SERVERS!` shouldn't be here.
There isn't an explicit dev vs release track separation for Citizens. The dev builds are often better than the spigot builds

Reply:
I added this because dev builds may be ahead of current release and be breaking...
So maybe change to "May have breaking changes"? Or "Always keep backups"?